#! /usr/bin/env python3
PACKAGE="collabot_do"
from dynamic_reconfigure.parameter_generator_catkin import *
gen=ParameterGenerator()

gen.add("ac_threshold",double_t,0,"ac_threshold",150.0,0.0,300.0)

exit(gen.generate(PACKAGE,"collabot_do","collabot_do"))